extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionServiceMessage.java

```diff
@@ -38,6 +39,9 @@
     public static final String RESPONSE_ADDRESS_HEADER = "X-Reply-Address";
     public static final String PROPERTY_MESSAGE_HEADERS = "messageHeaders";
 
+    // Set in parameters to allow reconnect after network half-open situation
+    public static final String RECONNECT_SECRET = "reconnectSecret";
```

Comment:
Should this be the "reconnect secret" or "connect secret"?

I think it's fine either way, but the secret would be used for both the first connection as well as any other reconnection afterwards.

Reply:
I'd prefer to leave it as is.  CONNECT_SECRET sounds like something that you need to connect (and, generally, you don't). It's also invisible to users of the SDK.

I'll think if there's a comment I can add that clarifies things.

Reply:
Works for me!